src/main/kotlin/no/nav/familie/ba/mottak/task/OppdaterOgFerdigstillJournalpostTask.kt

```diff
@@ -19,36 +19,40 @@ class OppdaterOgFerdigstillJournalpostTask(private val journalpostClient: Journa
                                            private val sakClient: SakClient,
                                            private val aktørClient: AktørClient,
                                            private val taskRepository: TaskRepository,
-                                           private val personClient: PersonClient) : AsyncTaskStep {
+                                           private val personClient: PersonClient,
+                                           private val infotrygdBarnetrygdClient: InfotrygdBarnetrygdClient) : AsyncTaskStep {
 
     val log: Logger = LoggerFactory.getLogger(OppdaterOgFerdigstillJournalpostTask::class.java)
 
     override fun doTask(task: Task) {
         val journalpost = journalpostClient.hentJournalpost(task.payload)
                                   .takeUnless { it.bruker == null } ?: throw error("Journalpost ${task.payload} mangler bruker")
         val brukersIdent = tilPersonIdent(journalpost.bruker!!)
-        val barnasIdenter = personClient.hentPersonMedRelasjoner(brukersIdent).familierelasjoner
+        val brukersIdenter = listOf(brukersIdent) // TODO: Legge til historiske identer
+        val barnasIdenter = personClient.hentPersonMedRelasjoner(brukersIdent).familierelasjoner // TODO: Legge til historiske identer
```

Comment:
Hvordan skal vi skaffe disse? Gjennopprette "hentIdenter" i familie-integrasjoner, eller opprette pdl-integrasjon i ba-mottak?

Reply:
Virker som om det er egen integrasjon som er tingen mot pdl. Kanskje det er mulig å slå de sammen med hentPerson kallet for relasjoner. Mener at graphql kan gjøre flere queries i et kall. uten at jeg har testet selv-

Reply:
altså direkte mot pdl